### PR TITLE
Issue 3366: Format issues fixed in documentation

### LINKED ISSUE #3366: Format Fix in Documentation
**Problem description**
The following format issues happened in the below listed documents when published in Pravega.io

**Controller-service.md**
1. Request flow and transaction flow diagram not rendered properly in browser after publication into to Pravega.io
2. In TOC the Section Controller Service had broken link

**Pravega-security-Authorization-authentication**
1. Listing not rendered properly

**Pravega-security-encryption:**
1. Typos

**Wire-protocol.md**
1. Listing not rendered properly in the last section

**Problem location**
documentation/controller-service.md
documentation /security/Pravega-security-Authorization -authentication.md
documentation /security/Pravega-security-encryption.md
documentation/wire-protocol.md

**Suggestions for improvement**
Fixed in website  and  master/documentation

### PR DESCRIPTION
**Pull request by Aparnarr**
**Change log description**  
Format issues related to Pravega.io were fixed in the following docs:

documentation/controller-service.md
documentation /security/Pravega-security-Authorization -authentication.md
documentation /security/Pravega-security-encryption.md
documentation/wire-protocol.md

**Purpose of the change**  
( Fixes #3366, Closes #3366)

**What the code does**  
Format fixed

### CODE REVIEW

**Comment by Aparnarr** [comment]
@fpj @RaulGracia please merge into master

File: documentation/src/docs/controller-service.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ interface. This includes API calls for Stream management as well as
 other administration API primarily dealing with _creation_ and _deletion_ of
 [**Scopes**](pravega-concepts.md#streams). We use [swagger](https://swagger.io) to describe our `REST` API. Please see, the swagger [`yaml`](https://github.com/pravega/pravega/tree/master/shared/controller-api/src/main/swagger) file.
 
-## Pravega Controller Service
+## Pravega Controller Service <a name ="controller-service"></a>
```
Comment (tkaitchuck) [inline documentation/src/docs/controller-service.md]: Why is there nothing inside of this link tag?

Comment (Aparnarr) [reply documentation/src/docs/controller-service.md]: @tkaitchuck @shiveshr As we had two sections having the title `Pravega controller service`....mismatch happened and the link was not rendered correctly so i included html tag for one of the section.... problem resolved.

**Comment by Aparnarr** [comment]
@fpj Please merge this PR into master.

**Comment by Aparnarr** [comment]
@fpj Please merge this into master

**Pull request merged**